### ISSUE TITLE
Incorrect error for relative import

### ISSUE THREAD

**Issue by steinybot** [joshuajaco/eslint-plugin-workspaces#11]
I have the following modules:
```
root/
child/
├─ foo/
```

In `child/foo/index.ts` if I try:
```
import {x} from "root";
```

Then eslint fails with:
```
❯ yarn run fix
yarn run v1.22.10
$ yarn eslint . --ext .js,.jsx,.ts,.tsx --fix
$ /Users/jason/source/bug-reports/node_modules/.bin/eslint . --ext .js,.jsx,.ts,.tsx --fix

/Users/jason/source/bug-reports/child/foo/index.ts
  1:1  error  Relative imports of other packages are not allowed  workspaces/no-relative-imports

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Checkout the branch here for a reproduction: https://github.com/steinybot/bug-reports/tree/eslint-plugin-workspaces/relative-import

**Comment by joshuajaco** [comment]
Fix published in `v0.7.0`
